src/components/SchedulePage/Timeslot.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import Event from '../SchedulePage/Event';
+import ScheduleRoundedIcon from '@material-ui/icons/ScheduleRounded';
+
+const useStyles = makeStyles(theme => ({
+	time: {
+		fontFamily: theme.typography.fontFamily,
+		marginLeft: 4
+	},
+	timeContainer: {
```

Comment:
Here's a good place so look for review. I'm not great with CSS, so I'm unsure if my styling is good here.

Reply:
This looks fine to me, but I'm also not the best with CSS.

Reply:
Instead of using flexboxes directly, can you instead use the [`<Box>`](https://material-ui.com/system/flexbox/#flexbox) component?